docs/development/design/apiv3.rst

```diff
@@ -1,15 +1,34 @@
-Design of APIv3
-===============
+APIv3
+=====
+
+This document describes the design,
+some decisions already made and an implementation plan for APIv3 in stages.
```

Comment:
This sentence needs something connecting the 2 halves.

Reply:
I'm not sure to understand what you refers to here. I expanded a little more the sentence, but I don't think that is what you want though :)